
def generate_arithmetic_exception():
    result = 10 / 0  # This will cause a ZeroDivisionError

generate_arithmetic_exception()
